src/peerconnection.cc

```diff
@@ -53,15 +50,11 @@ PeerConnection::PeerConnection(webrtc::PeerConnectionInterface::IceServers iceSe
   webrtc::PeerConnectionInterface::RTCConfiguration configuration;
   configuration.servers = iceServerList;
 
-  webrtc::FakeConstraints constraints;
-  constraints.AddOptional(webrtc::MediaConstraintsInterface::kEnableDtlsSrtp, webrtc::MediaConstraintsInterface::kValueTrue);
-  // FIXME: crashes without these constraints, why?
-  constraints.AddMandatory(webrtc::MediaConstraintsInterface::kOfferToReceiveAudio, webrtc::MediaConstraintsInterface::kValueFalse);
-  constraints.AddMandatory(webrtc::MediaConstraintsInterface::kOfferToReceiveVideo, webrtc::MediaConstraintsInterface::kValueFalse);
+  // TODO(mroberts): Read `factory` (non-standard) from RTCConfiguration?
+  _factory = PeerConnectionFactory::GetOrCreateDefault();
+  _shouldReleaseFactory = true;
```

Comment:
In the future, it might make sense to allow

```js
const { RTCPeerConnection, RTCPeerConnectionFactory } = require('wrtc')

const factory = new RTCPeerConnectionFactory({
  audioLayer: 'dummy' // or any of the other AudioLayers
})

const pc = new RTCPeerConnection({
  factory // non-standard addition to RTCConfiguration
})
```

This would allow experimenting with different AudioDeviceModules, as well as control over threads (since there is one signaling and one worker thread per RTCPeerConnectionFactory). Not sure if more threads == better, but it would be an option. Also, for reference, the AudioLayers are

```cpp
enum AudioLayer {
  kPlatformDefaultAudio = 0,
  kWindowsWaveAudio = 1,
  kWindowsCoreAudio = 2,
  kLinuxAlsaAudio = 3,
  kLinuxPulseAudio = 4,
  kAndroidJavaAudio = 5,
  kAndroidOpenSLESAudio = 6,
  kAndroidJavaInputAndOpenSLESOutputAudio = 7,
  kDummyAudio = 8
};
```